python/cuml/tests/test_qn.py

```diff
@@ -75,48 +75,42 @@ def test_qn(loss, dtype, penalty, l1_strength, l2_strength, fit_intercept):
                 assert (qn.objective - 0.40263831615448) < tol
                 cp.testing.assert_array_almost_equal(
                     qn.coef_,
-                    np.array([[-2.1088872],
-                              [2.4812558]]),
+                    np.array([[-2.1088872, 2.4812558]]),
```

Comment:
Why has this behavior changed?

Reply:
Coefficients in cuML were transposed relative to sklearn. Some of the test files had to be modified to account for the change.